packages/mock-service/src/controllers/tweets.ts

```diff
@@ -1,6 +1,11 @@
 import { tweetsData } from "../../tweetsData";
 import { Request, Response } from "express";
 
-export const TweetsController = (_: Request, res: Response) => {
+export const GetTweetsController = (_: Request, res: Response) => {
   res.send(tweetsData);
-};
\ No newline at end of file
+};
+
+export const PostTweetController = (req: Request, res: Response) => {
+  tweetsData.push(req.body);
+  res.send(tweetsData);
+};
```

Comment:
Usually, in MVC pattern, a controller holds all the route handling for a particular resource, so you would see a class/module called `TweetController` for example, and it would encapsulate the logic to handle GET, POST, etc requests.

There's nothing wrong with this approach you have in the code, but since the module itself is a controller, rathe than exporting multiple route handlers, exporting a single route handler already configured to handle all `/tweets` resources operations may be a more "controller"-like approach.

Here's a JS example (you may need to figure out how to translate it to TS)

```ts
// tweets.controller.ts
const express = require('express');

const router = express.Router();

router.get('/', (req, res, next) => {
  // handle GET /tweets request
});

router.post('/', (req, res, next) => {
  // handle POST /tweets request
});

module.exports = router;
```

```js
// app.js
app.use('/tweets', tweetController);  // routes all `/tweet` URI matches to the Tweet controller
```